Source/AGXUnreal/Public/AGX_Simulation.h

```diff
@@ -343,9 +343,17 @@ class AGXUNREAL_API UAGX_Simulation : public UGameInstanceSubsystem
 	 * Whether or not to override each AGX Shape with the selected AdditionalUnrealCollision
 	 * setting. If this is set to false, the setting on each AGX Shape is used instead.
 	 */
-	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Other")
+	UPROPERTY(Config, EditAnywhere, BlueprintReadOnly, Category = "Other")
 	bool bOverrideAdditionalUnrealCollision {false};
 
+	/**
+	 * Keyboard shortcut to enable Grab Mode, making it possible to grab a Rigid Body in the
+	 * viewport with the mouse.
+	 * Note: if changed, the Unreal Editor must be restarted for the change to take effect.
```

Comment:
The reason is because of Module dependency directions. 
Whenever we change this setting, we are making changes to the `AGXUnreal` module. 
But to re-bind the command, we need to be notified in the `AGXUnrealEditor` module, which we cannot reach from UAGX_Simulation. 

Doing work-around with a delegate seems a bit hacky. Thoughts?